mitmproxy/tools/main.py

```diff
@@ -34,134 +35,80 @@ def assert_utf8_env():
         sys.exit(1)
 
 
-def process_options(parser, options, args):
+def process_options(parser, opts, args):
     if args.version:
         print(debug.dump_system_info())
         sys.exit(0)
+    if args.options:
+        print(optmanager.dump(opts))
+        sys.exit(0)
+    if args.quiet:
```

Comment:
Any reason why we special-case quiet? Seems like we could just define the command line arg as store_const 0.